update-static-saddlebag-data.py

```
@@ -0,0 +1,54 @@
+#!/usr/bin/python3
+import json, requests
+from utils.api_requests import (
+    get_itemnames,
+    get_ilvl_items,
```

Ruff

`utils.api_requests.get_ilvl_items` imported but unused

Remove unused import: `utils.api_requests.get_ilvl_items`

(F401)